File: src/main/java/io/github/vehkiya/data/model/json/JsonSource.java
package io.github.vehkiya.data.model.json;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

import java.util.Map;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class JsonSource {

    private Map<String, JsonItem> items;
}

File: src/main/java/io/github/vehkiya/data/model/json/JsonSource.kt
package io.github.vehkiya.data.model.json

import com.fasterxml.jackson.annotation.JsonAutoDetect
import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
data class JsonSource(
    val items: Map<String, JsonItem>
)
